### ISSUE TITLE
visual error with sophisticated storage and backpacks mod 

### ISSUE THREAD

**Issue by Sparky2015** [EnigmaticaModpacks/Enigmatica10#297]
### Modpack Version

1.21.1

### Describe your issue.

when using any of the sophisticated storage or backpacks there is a visual issue that makes it when your using either it makes it all super dark when in the their corresponding gui and makes it hard too see what your looking at in while using the mod

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

### Discord Username

airbourneoutlaw

**Comment by MuteTiefling** [comment]
Thanks for the report!

Is it possible you have dark mode everywhere enabled? Also, could you post a screenshot?

**Comment by Sparky2015** [comment]
i dont have dark mode on but its now happening to the market block but im pretty sure its happening to other blocks as well
![image](https://github.com/user-attachments/assets/dd980758-d53e-4249-a6f6-2b4c19ae323e)

**Comment by Sparky2015** [comment]
I'm just running the base modpack and i went and made duplicate save of the profile and reopened the modpack again and its now happening to just dire things 
![image](https://github.com/user-attachments/assets/437e74ad-4e02-4f6b-b1b4-d256ce021806)

**Comment by Sparky2015** [comment]
now i just see a bunch of darkness where i look
![image](https://github.com/user-attachments/assets/650266a4-04a3-47e3-bbfb-f3e0921db083)

**Comment by MuteTiefling** [comment]
Yeah, I think you should bring this to our support channel in Discord. This isn't a pack bug. Seems more like an issue with your PC. We'll be able to help you better there.

**Comment by Sparky2015** [comment]
i havent used any of the ars portal it will just randomly happen whilst im playing the modpack with my friend and they have had a few things happen to them as well while we have been playing

**Comment by MuteTiefling** [comment]
Yeah, there have been a few more reports on Discord as well. 

What we know so far
1. It happens to certain players repeatedly and others never
2. It can impact a player on a server without impacting others, so it's very much client sided
3. It has a broader reach than just this pack, other 1.21 packs are also impacted. 

As a result of all this, my gut instinct is that it's still something at the PC level that only certain people have; drivers or some particular Windows update for instance. Whatever it is is not playing nice with minecraft/neoforge/some-particular-mod.

I also haven't seen anything similar in Dev so I either just don't meet the criteria for it to break, or someone somewhere has already fixed it (lots of mods updated in dev).

If we're going to narrow this down, we need people who can reliably reproduce the issue to find commonalities between their machines; when it started, hardware, updates installed around the time it started, etc. 

And if there are none, we then need someone who can reproduce it to start disabling mods until it goes away for them. It may make sense to start with things like Sodium/Iris that have no gameplay impact (ie, removing them isn't going to destroy your world because blocks are missing) but anything that touches rendering could potentially be related.

**Comment by dravster** [comment]
I'm running a similar set of mods in my own private modpack and seeing similar issues, as are friends joining the server. Hardware List are:

AMD Ryzen 7 3800X / Nvidia RTX 3060-Ti
Intel i9  i9-9900K / Nvidia GeForce RTX 2080 Super
AMD Ryzen 7950x3d / AMD RX 7900 XTX

When I get back after the xmas break I'm going to have a go at spinning up the server with some of the mods removed (including Sodium) to see if I can narrow down the combination that causes the issue.

**Comment by MuteTiefling** [comment]
With the release of 1.16, and the fact that apparently nobody with this issue has even remotely similar hardware, I'd ask that everyone that was experiencing this update to 1.16 and see if the problem persists.

**Comment by jjjworley** [comment]
This isn't the only modpack it's happened on. I also played All the mods 10 and FTB Evolution and the same thing happened on those modpacks. I don't know if it happens in single player, I've only played on servers when it happens. VERY ANNOYING!!!
